homeassistant/components/repetier/sensor.py

```diff
@@ -169,7 +169,7 @@ def update(self):
         print_time = data["print_time"]
         from_start = data["from_start"]
         time_end = start + round(print_time, 0)
-        self._state = datetime.utcfromtimestamp(time_end)
+        self._state = dt_util.utc_from_timestamp(time_end).replace(tzinfo=None)
```

Comment:
Unsure if we could work with aware datetime here.

Reply:
This sensor has the `SensorDeviceClass.TIMESTAMP` device class, meaning removing the TZ will break it (we require it to have a TZ).

Reply:
Ok, leaving as is, ditto other suggestions you replied to below. Thanks!

(Conflicts resolved again, too.)

Reply:
I don't understand the conclusion here. Normally, timestamp sensors should return a local aware datetime. It will be converted in the base sensor class to a UTC timestamp.

Reply:
Are you saying we'll keep it broken in this PR?

Reply:
Yeah, some confusion here indeed. What was implemented here was not removing the timezone in the sense that the timezone wasn't there before, either, I took that was what frenck meant above. Made it aware in f70887bec3f0c8abc9eb17f539d6e243cdadd0d7.

Reply:
(Kept it a UTC one because that's what it was, and a small sample of other timestamp sensors I took a peek at use aware UTC timestamps, too.)

Reply:
Yeah the timezone should just be consistent with the api in the integration. If the integration wants to use UTC that's ok.